qcodes_contrib_drivers/drivers/ERAInstruments/erasynth.py

```diff
@@ -0,0 +1,900 @@
+
+        # WiFi control, NB initial_cache_value is used to avoid overriding these values
+
+        self.wifi_mode = Parameter(
+            name="wifi_mode",
+            instrument=self,
+            val_mapping={"station": "0", "hotspot": "1", "": ""},
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEW']}",
+            set_cmd="PEW{}",
+        )
+        """WiFi Mode, station/hotspot."""
+
+        self.wifi_station_ssid = Parameter(
+            name="wifi_station_ssid",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PES0']}",
+            set_cmd="PES0{}",
+        )
+        """Sets network SSID for WiFi module."""
+
+        self.wifi_station_password = Parameter(
+            name="wifi_station_password",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEP0']}",
+            set_cmd="PEP0{}",
+        )
+        """Sets network password for WiFi module."""
+
+        self.wifi_hotspot_ssid = Parameter(
+            name="wifi_hotspot_ssid",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PES1']}",
+            set_cmd="PES1{}",
+        )
+        """Sets hotspot SSID for WiFi module."""
+
+        self.wifi_hotspot_password = Parameter(
+            name="wifi_hotspot_password",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEP1']}",
+            set_cmd="PEP1{}",
+        )
+        """Sets hotspot password for WiFi module."""
+
+        self.wifi_ip_address = Parameter(
+            name="wifi_ip_address",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEI']}",
+            set_cmd="PEI{}",
+        )
+        """Sets IP address for WiFi module."""
+
+        self.wifi_subnet_address = Parameter(
+            name="wifi_subnet_address",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEN']}",
+            set_cmd="PEN{}",
+        )
+        """Sets Subnet mask for WiFi module."""
+
+        self.wifi_gateway_address = Parameter(
+            name="wifi_gateway_address",
+            instrument=self,
+            vals=validators.Strings(),
+            get_cmd=f"RA:{_CMD_TO_JSON_MAPPING['PEG']}",
+            set_cmd="PEG{}",
+        )
+        """Sets default gateway for WiFi module."""
+
+        self.debug_messages_en = Parameter(
+            name="debug_messages_en",
+            instrument=self,
+            vals=validators.Strings(),
+            initial_cache_value=None,
+            val_mapping={True: "1", False: "0"},
+            set_cmd="PD{}",
+        )
+        """Enables/disables debug printing on the serial port."""
+
+        self.visa_handle.baud_rate = BAUDRATE
```

Comment:
You are trying to add `baud_rate` attribute to `visa_handle` and set it to `BAUDRATE`. 
How can this be useful?
This is also one of mypy complains.

Reply:
I worked in this part a while ago, and do not fully recall this, I believe the idea was to set https://pyvisa.readthedocs.io/en/latest/api/resources.html?highlight=baud_rate#pyvisa.resources.SerialInstrument.baud_rate

Reply:
OK. I wanted to make sure if everything is correct in the run time, then we can look into mypy thing